logs/logs.go

```diff
@@ -0,0 +1,65 @@
+package logs
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	"github.com/rancherio/host-api/app/common/connect"
+	"github.com/rancherio/host-api/config"
+	"net/http"
+	"strconv"
+)
+
+func GetLogs(rw http.ResponseWriter, req *http.Request) error {
+	urlToken := req.URL.Query().Get("token")
+	token, err := jwt.Parse(urlToken, func(token *jwt.Token) (interface{}, error) {
+		return token, nil
```

Comment:
I think you are mistaken.... You return the token but pass it generally to a function that tests for the key with in the token to decide it if it is returned or errors. At least from my understanding of the docs